src/Simulation/Simulators.Tests/QuantumSimulatorTests/VerifyGates.cs

```diff
@@ -174,7 +204,13 @@ public void QSimVerifyS()
         [Fact]
         public void QSimVerifyT()
         {
-            using (var sim = new QuantumSimulator())
+            var simulators = new CommonNativeSimulator[] { 
+                new QuantumSimulator(),
+                //new SparseSimulator2()    // TODO(rokuzmin): Uncomment once the MCT is implemented.
```

Comment:
Is this TODO bottlenecked on decompositions, or is the multi-controlled `T` expected to be supported directly within the sparse simulator?

Reply:
Uncommenting after the recent Thomas' commits implementing MCT.